python_src/src/lamp_py/runtime_utils/import_env.py

```diff
@@ -11,7 +11,7 @@ def load_environment() -> None:
             return
 
         here = os.path.dirname(os.path.abspath(__file__))
-        env_file = os.path.join(here, "..", "..", "..", ".env")
+        env_file = os.path.join(here, "..", "..", "..", "..", ".env")
```

Comment:
Just need an extra set of ".." here because this got moved into the `runtime_utils` folder.